library/src/lib/time-picker/time-picker.component.ts

```diff
@@ -21,44 +21,64 @@ import { PopperOptions } from 'popper.js';
     styles: [':host {display: inline-block;}']
 })
 export class TimePickerComponent implements ControlValueAccessor, OnInit {
+
+    /** @Input An object that contains three integer properties: 'hour' (ranging from 0 to 23),
+     * 'minute' (ranging from 0 to 59), and 'second' (ranging from 0 to 59).  */
     @Input()
     time: TimeObject = { hour: 0, minute: 0, second: 0 };
```

Comment:
As mentioned in the other PR, consider using markdown to make this look a bit nicer, maybe a list or something?

also maybe explain what it is used for on top of what it consists of